token/program-2022/src/extension/mod.rs

```diff
@@ -142,7 +160,12 @@ fn get_extension_indices<V: Extension>(
     Err(ProgramError::InvalidAccountData)
 }
 
-fn get_extension_types(tlv_data: &[u8]) -> Result<Vec<ExtensionType>, ProgramError> {
+/// Fetches the extension types written to the buffer, along with the total
+/// number of bytes used by the TLV entries, including the type and length of
+/// each extension.
```

Comment:
The punctuation makes this a little ambiguous; one way to read it is: "Fetches the extension types written to the buffer, including the type and length of each extension."

The best rewrite I've come up with so far is below, but I'm not totally happy with it. Got any ideas?

```suggestion
/// Fetches the extension types written to the buffer, along with the total
/// number of bytes used by the TLV entries, each of which includes the type and length of
/// the extension.
```

Reply:
When adding the new type, I added this info more clearly into its field comment. Let me know how it looks!